File: rest/taskrouter/workspaces/list/post/example-1/example-1.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "{{ account_sid }}";
    string AuthToken = "{{ auth_token }}";
    string WorkspaceSid = "{{ workspace_sid }}";
    var client = new TaskRouterClient(AccountSid, AuthToken);

    Workspace workspace = client.AddWorkspace("NewWorkspace", "http://requestb.in/vh9reovh", "FIFO");
    Console.WriteLine(workspace.FriendlyName);
  }
}

File: rest/taskrouter/taskqueues/list/get/example-1/example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioTaskRouterClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "{{ account_sid }}"
auth_token  = "{{ auth_token }}"
workspace_sid = "{{ workspace_sid }}"

client = TwilioTaskRouterClient(account_sid, auth_token)

for taskqueue in client.taskqueues(workspace_sid).list():
	print taskqueue.friendly_name